blank-canvas-blocks/experimental-theme.json

```diff
@@ -144,7 +144,17 @@
 					"color": "#ccc",
 					"thickness": "2px",
 					"width": "150px"
+				},
+				"list": {
+					"fontFamily": "var(--wp--custom--font-family--base)",
+					"padding": {
+						"top": 0,
```

Comment:
If these values aren't used by the styles, should we be creating custom variables for them?